src/content/advanced/egress-ip-address-azure/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 linkTitle: Egress IP address on Azure
-title: Egress IP address on Azure
-description: How to reuse an existing Public IP address for egress traffic of worker nodes on Azure.
+title: Setting an egress IP address on Azure
+description: How to reuse an existing public IP address for outgoing traffic of worker nodes on Azure.
```

Comment:
Using different wording than "egress" here might help make it easier to understand.